### PR TITLE
[pylint] fixing enum checker

### PR DESCRIPTION
**Pull request by l0lawrence**
Only str enum classes inherit metaclass

### CODE REVIEW

File: scripts/pylint_custom_plugin/pylint_guidelines_checker.py [hunk 1]
```diff
@@ -1763,7 +1763,7 @@ def visit_classdef(self, node):
                     self._enum_uppercase(node)   
             # Else if it does not have a metaclass, but it is an enum class
             # Check both capitalization and throw pylint error for metaclass
-            elif node.bases[1].name == "Enum":
+             elif node.bases[0].name == "str" and node.bases[1].name == "Enum":
```
Comment (kristapratico) [inline scripts/pylint_custom_plugin/pylint_guidelines_checker.py:1766]: seems like this line starts with an extra space that's not needed?

**Review by kristapratico — APPROVED** [review]

File: scripts/pylint_custom_plugin/pylint_guidelines_checker.py (hunk 1 above)
Comment (catalinaperalta) [inline scripts/pylint_custom_plugin/pylint_guidelines_checker.py:1766]: Nit: extra space

**Review by catalinaperalta — APPROVED** [review]

**Pull request merged**